### PR TITLE
update and fix boilerplate generation script

### LINKED ISSUE #925: Generate script uses a wrong path
### Describe the bug

The script does not use the correct path when only entering the component name e.g. `test`

### Describe how to produce the bug

Use a name for the component without any path information given.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #925

### CODE REVIEW

File: scripts/package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@helpwave/scripts",
+  "version": "1.0.0",
+  "description": "A collection of scripts used for programming with helpwave",
```
Comment (MaxSchaefer) [inline scripts/package.json:4]:
```suggestion
  "description": "A collection of custom tooling for helpwave/web.",
```

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
- Please provide some example commands in the `README`
- Error during execution
```shell
❯ node scripts/generate_boilerplate.js Username
node:internal/validators:162
    throw new ERR_INVALID_ARG_TYPE(name, 'string', value);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:387:5)
    at validateString (node:internal/validators:162:11)
    at Object.resolve (node:path:1098:7)
    at Object.<anonymous> (/Users/maxrs/Projects/helpwave/helpwave-web/scripts/generate_boilerplate.js:44:21)
    at Module._compile (node:internal/modules/cjs/loader:1165:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1219:10)
    at Module.load (node:internal/modules/cjs/loader:1043:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:22:47 {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

**Comment by DasProffi** [comment]
@MaxSchaefer same CI issue here and I have rewritten the script so that it should work

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
Runs now. Thanks!
`node scripts/generate_boilerplate.js -f -nop -notl tasks/components/Username`
`-notl` works fine. `-nop` should also remove the `UsernameProps` type.

```tsx
import { tw } from '@helpwave/common/twind'

export type UsernameProps = Record<string, never>

/**
 * Description
 */
export const Username = ({
} : UsernameProps) => {
  return (
    <div>
      { /* Your Code */ }
    </div>
  )
}

```

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**